static/src/js/actions/errors.js

```diff
@@ -0,0 +1,26 @@
+import uuidv4 from 'uuid/v4'
+
+import { REMOVE_ERROR, ADD_ERROR } from '../constants/actionTypes'
+
+export const addError = payload => ({
+  type: ADD_ERROR,
+  payload
+})
+
+export const removeError = payload => ({
+  type: REMOVE_ERROR,
+  payload
+})
+
+export const handleError = (error, resource, verb = 'retrieving') => (dispatch) => {
+  const {
+    error: message = 'There was a problem completing the request'
+  } = error
+
+  dispatch(addError({
+    id: uuidv4(),
+    title: `Error ${verb} ${resource}`,
+    message,
+    details: error
+  }))
```

Comment:
It might make sense to `console.error` here instead of in the actions. We're generating a UID here but we're never using it alongside our error messages. Ideally users can provide this UID to us when reporting errors (I know that would require we store them in the db or the error happen in Lambda, we'll get there). But, given that this is called from ALL of our actions, it seems like it'd make sense to move the logging here and include the UID in the message to get users used to seeing it.